doc/source/examples/admonitions.rst

```diff
@@ -0,0 +1,31 @@
+.. _admonitions:
+
+Admonitions
+===========
+
+Admonitions are a way to highlight important information in documentation.
+This page shows how the admonitions are rendered in the documentation with ``ansys sphinx theme``.
+
+
+.. jinja:: admonitions
+
+    {% for filename in inputs_admonitions %}
+
+    .. include:: sphinx_examples/{{ filename }}
+
+    {% endfor %}
+
+Credits and copyright
```

Comment:
Does this section and child section need to exist. It seems that the information can be provided as part of the Admonitions section, without headings?

Reply:
sure, we can add that